src/iomei.cpp

```diff
@@ -1589,6 +1607,10 @@ void MEIOutput::WriteMdiv(pugi::xml_node currentNode, Mdiv *mdiv)
 {
     assert(mdiv);
 
+    if (mdiv->IsHidden() && this->IsSerializing()) {
+        m_currentNode.append_attribute("verovio.serialization") = "hidden";
```

Comment:
Maybe we can define `verovio.serialization` as a string constant somewhere. It is used a lot.

Reply:
I'll have a define